TLM/TLM/Util/Extensions/VehicleExtensions.cs

```diff
@@ -9,5 +20,29 @@ public static class VehicleExtensions {
             vehicle.m_flags.CheckFlags(
                 required: Vehicle.Flags.Created,
                 forbidden: Vehicle.Flags.Deleted);
+
+        /// <summary>Queues the vehicle to be despawned.</summary>
+        /// <param name="vehicle">The vehicle instance to despawn.</param>
+        // TODO: Would vehicle.Unspawn() be viable/better approach?
+        public static void Despawn(this Vehicle vehicle) =>
```

Comment:
Bad idea, it will spam Action list with thousands of actions. Maybe pass list of ID to despawn?

Reply:
I could use `VehicleManager::ReleaseVehicle` like the prior `ClearTraffic()` method?

Reply:
If the `.Despawn` extension is used only for individual vehicles, will it suffice to leave it as is?

I can alter the `DespawnVehicles(filter..)` so it calls `VehicleManager::ReleaseVehicle` instead of the `.Despawn()` extension?

Reply:
Pushed a commit that makes the `DespawnVehicles()` use `ReleaseVehicle`; added a `<remark>` to `.Despawn` xmldoc mentioning that it shouldn't be used for large numbers of vehicles.

I could still update `.Despawn()` to use `ReleaseVehicles()` - lmk what you think is best.

Reply:
removed extension as it is no longer used